app/web/features/communities/discussions/CommentTree.tsx

```diff
@@ -1,57 +1,33 @@
-import { Typography } from "@mui/material";
+import { styled, Typography } from "@mui/material";
 import Alert from "components/Alert";
 import Button from "components/Button";
 import CenteredSpinner from "components/CenteredSpinner/CenteredSpinner";
 import { useTranslation } from "i18n";
 import { COMMUNITIES } from "i18n/namespaces";
+import { theme } from "theme";
 import hasAtLeastOnePage from "utils/hasAtLeastOnePage";
-import makeStyles from "utils/makeStyles";
 
 import { useThread } from "../hooks";
 import Comment from "./Comment";
 import CommentForm from "./CommentForm";
 
-const useStyles = makeStyles((theme) => ({
-  commentsListContainer: {
-    "& > * + *": {
-      marginBlockStart: theme.spacing(2),
-    },
-    display: "flex",
-    flexDirection: "column",
+const StyledCommentsListContainer = styled("div")(({ theme }) => ({
+  "& > * + *": {
     marginBlockStart: theme.spacing(2),
-    marginBlockEnd: theme.spacing(6),
-    [theme.breakpoints.down("sm")]: {
-      //break out of page padding
-      left: "50%",
-      marginLeft: "-50vw",
-      marginRight: "-50vw",
-      position: "relative",
-      right: "50%",
-      width: "100vw",
-    },
   },
-  loadEarlierCommentsButton: {
-    alignSelf: "center",
-  },
-  commentContainer: {
-    display: "flex",
-    padding: theme.spacing(2),
-    width: "100%",
-  },
-  commentContent: {
-    "& > * + *": {
-      marginBlockStart: theme.spacing(0.5),
-    },
-    display: "flex",
-    flexDirection: "column",
-    marginInlineStart: theme.spacing(3),
-  },
-  noComment: {
-    marginBlockEnd: theme.spacing(6),
-  },
-  avatar: {
-    height: "3rem",
-    width: "3rem",
+  padding: theme.spacing(0, 2),
```

Comment:
This line is the only thing that changed here to make it look better on mobile. The rest is converting to styled components.